package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scripts": {
+    "build": "tsc",
+    "start": "npx live-server"
```

Comment:
Why doing `npx` if this package is already in your repo?
1. Read what is npx
2. Change just to `live-server`

Reply:
If you will try to add any changes to your app you will notice how developer experience sucks, because after each change you'll need to do `build` and `start`. 

Add `watch` script so at least one thing that will be needed is page reload